lib/ansible/modules/cloud/amazon/elb_target_group_facts.py

```diff
@@ -146,6 +153,22 @@
             returned: always
             type: str
             sample: "arn:aws:elasticloadbalancing:ap-southeast-2:01234567890:targetgroup/mytargetgroup/aabbccddee0044332211"
+        targets_health_description:
+            description: The targets health of the target group.
+            returned: when collect_targets_health is enabled
+            type: dict
+            sample: "[
```

Comment:
This looks like it conflicts with the code - you seem to be doing camel to snake on the health description (which is good) so the sample should also be in snake case.

Also, if you use hacking/return_skeleton_generator.py, it will do a better job of documenting all of the fields in the target health (it might be overkill in this case)

Reply:
Thank you for the review!
I fixed case issue and documented all nested fields with help of hacking/return_skeleton_generator.py